### PR TITLE
Update UpdateOccupationsWithNOC2021.sql

### PR DESCRIPTION
**Pull request by sunanditasahu**

### CODE REVIEW

File: SSOT/UpdateOccupationsWithNOC2021.sql [hunk 1]
```diff
@@ -7,7 +7,7 @@ FROM OPENJSON (@JSON)
 WITH 
 (
     [noc_2021] varchar(10), 
-    [label] varchar(50) 
+    [label] varchar(500)
```
Comment (infojunkie) [inline SSOT/UpdateOccupationsWithNOC2021.sql:10]: This needs to be the exact same type as the destination column.

**Review by infojunkie — CHANGES_REQUESTED** [review]
Left a comment

File: SSOT/UpdateOccupationsWithNOC2021.sql (hunk 1 above)
Comment (sunanditasahu) [reply SSOT/UpdateOccupationsWithNOC2021.sql]: Done the change

**Review by infojunkie — APPROVED** [review]
Thanks!

**Pull request merged**